requirements.txt

```diff
@@ -256,3 +256,5 @@ WTForms==2.1 \
     --hash=sha256:ffdf10bd1fa565b8233380cb77a304cd36fd55c73023e91d4b803c96bc11d46f
 Flask_WTF==0.14.2 \
     --hash=sha256:d9a9e366b32dcbb98ef17228e76be15702cd2600675668bca23f63a7947fd5ac
+newrelic==2.78.0.57 \
```

Comment:
I don't think it works like that. the other file is only used for unit / integration tests?
This in theory isn't a test dependency as we want it in the code all the time.